<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trình trình sửa trường hàng loạt</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/logo/icon-sw.png</normaloff>:/logo/icon-sw.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="bulkEditTitle">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bạn muốn áp dụng thay đổi với trường như thế nào?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QScrollArea{
border:0
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>825</width>
            <height>501</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="aboveSection">
             <property name="styleSheet">
              <string notr="true">QGroupBox{
    border: 1px solid #a4b0be;
    margin-top: 0.5em;
	font-weight:bold
}
QGroupBox::title {
    top: -6px;
    left: 10px;
}
</string>
             </property>
             <property name="title">
              <string>1 cho tất cả</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="topMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tiêu đề đó là gì? &lt;span style=&quot; color:#7c7c7c;&quot;&gt;(Nếu bạn sử dụng tăng dần có nghĩa là nội dung thêm vào + bắt đầu từ tự tăng dần, ví dụ part 1, part#1,..)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <property name="spacing">
                 <number>5</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="oneTitle"/>
                </item>
                <item>
                 <widget class="QRadioButton" name="keepOriginal">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Giữ nguyên</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="increaseNumber">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Tăng dần</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="increaseSection" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>nội dung thêm vào</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="fixedText">
                     <property name="maximumSize">
                      <size>
                       <width>70</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>part</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>bắt đầu từ</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="startFrom">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>20</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="belowSection">
             <property name="styleSheet">
              <string notr="true">QGroupBox{
    border: 1px solid #a4b0be;
    margin-top: 0.5em;
	font-weight:bold
}
QGroupBox::title {
    top: -6px;
    left: 10px;
}
</string>
             </property>
             <property name="title">
              <string>Rải theo số lượng</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="topMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nhập tất cả tiêu đề bạn có vào đây và chọn kiểu rải. &lt;span style=&quot; color:#ff1f0f;&quot;&gt;Mỗi 1 dòng được coi là 1 tiêu đề&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QRadioButton" name="ordered">
                  <property name="text">
                   <string>từ trên xuống</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="random">
                  <property name="text">
                   <string>ngẫu nhiên</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>400</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="thirdSection">
             <property name="styleSheet">
              <string notr="true">QGroupBox{
    border: 1px solid #a4b0be;
    margin-top: 0.5em;
	font-weight:bold
}
QGroupBox::title {
    top: -6px;
    left: 10px;
}
</string>
             </property>
             <property name="title">
              <string>Thêm hàng loạt vào vị trí nhắm đến</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="topMargin">
               <number>20</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QRadioButton" name="startOfTitle">
                  <property name="text">
                   <string>Đầu tiêu đề</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="endOfTitle">
                  <property name="text">
                   <string>cuối tiêu đề</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bạn có thể thêm hashtag, text, hay bất cứ thứ gì tuỳ ý mình muốn tại đây.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLineEdit" name="textNeedToAdd"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="fourthSection">
             <property name="styleSheet">
              <string notr="true">QGroupBox{
    border: 1px solid #a4b0be;
    margin-top: 0.5em;
	font-weight:bold
}
QGroupBox::title {
    top: -6px;
    left: 10px;
}
</string>
             </property>
             <property name="title">
              <string>Tìm kiếm và thay thế toàn bộ nội dung nhắm đến</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="topMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#7c7c7c;&quot;&gt;(Nếu bạn để trống trường &lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;biến thành&lt;/span&gt;&lt;span style=&quot; color:#7c7c7c;&quot;&gt; có nghĩa là bạn sẽ xoá chuỗi ký tự cần thay thế mà bạn nhập vào)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Chuỗi ký tự cần thay thế</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="replaceThisText"/>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>biến thành</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="replaceToThis"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Bạn đang chuẩn bị áp dụng thay đổi cho các video ở các dòng dưới đây trên bảng chính:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Áp dụng vào</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="toTitle">
              <property name="text">
               <string>tiêu đề</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="toDescription">
              <property name="text">
               <string>mô tả</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="toTags">
              <property name="text">
               <string>thẻ tags</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="whichIndex">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="applyToTable">
           <property name="text">
            <string>Áp dụng vào bảng</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearAll">
           <property name="text">
            <string>Xoá sạch tiêu đề</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="undoAction">
           <property name="text">
            <string>Hoàn lại thao tác vừa xong</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
